exonum-light-client/src/test/java/com/exonum/client/ExonumHttpClientBlocksIntegrationTest.java

```diff
@@ -45,28 +53,40 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import com.exonum.binding.common.hash.HashCode;
+import com.exonum.client.ExplorerApiHelper.GetBlockResponseBlock;
 import com.exonum.client.request.BlockFilteringOption;
 import com.exonum.client.request.BlockTimeOption;
 import com.exonum.client.response.Block;
 import com.exonum.client.response.BlockResponse;
-import com.exonum.client.response.BlocksResponse;
+import com.exonum.client.response.BlocksRange;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Lists;
 import java.io.IOException;
+import java.util.List;
 import java.util.Optional;
+import java.util.stream.LongStream;
 import okhttp3.mockwebserver.MockResponse;
 import okhttp3.mockwebserver.MockWebServer;
 import okhttp3.mockwebserver.RecordedRequest;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
 import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
 import org.junit.jupiter.params.provider.ValueSource;
 
+@Execution(ExecutionMode.SAME_THREAD)
 class ExonumHttpClientBlocksIntegrationTest {
-  private static MockWebServer server;
-  private static ExonumClient exonumClient;
 
-  @BeforeAll
-  static void start() throws IOException {
+  private MockWebServer server;
+  private ExonumClient exonumClient;
+
+  @BeforeEach
```

Comment:
it takes some time to start the server...

Reply:
First, it is quick enough (tens of ms even for tests requesting thousands of blocks), second — it does not matter at all when one breaking test breaks the other, unrelated, **that otherwise works** (it happened when I started coding with the tests and broken impl)